script/script.js

```diff
@@ -184,6 +187,11 @@ var Vector2 = (function() {
         ctx.stroke();
     };
 
+    // Send packet to set player's direction on a delay, to avoid running in to rate-limit.
+    setInterval(function() {
+        sendPacket(targetDirection);
+    }, 55);
+
```

Comment:
I see, you must have a better understanding of what's causing the ban, good work! 👍